packages/common/src/index.ts

```diff
@@ -196,6 +197,29 @@ export default class Common {
     return value
   }
 
+  /**
+   * Returns a parameter corresponding to an EIP
+   * @param topic Parameter topic ('gasConfig', 'gasPrices', 'vm', 'pow')
+   * @param name Parameter name (e.g. 'minGasLimit' for 'gasConfig' topic)
+   * @param hardfork Name of the EIP (e.g. 'EIP2537')
+   */
+  paramByEIP(topic: string, name: string, eip: string): any {
+    if (!(eip in EIPs)) {
+      throw new Error(`${eip} not supported`)
+    }
+
+    let value
```

Comment:
Nit: unnecessary variable declaration

Reply:
Done (thanks, please really point these things out, since these things are just unnecessarily complicating the code base)